docs/public/SUMMARY.md

```diff
@@ -0,0 +1,121 @@
+- [Installation](installation/README.md)
+    - [Prerequisites](installation/prerequisites.md)
+    - [Zonemaster-LDNS](installation/zonemaster-ldns.md)
+    - [Zonemaster-Engine](installation/zonemaster-engine.md)
+    - [Zonemaster-CLI](installation/zonemaster-cli.md)
+    - [Zonemaster-Backend](installation/zonemaster-backend.md)
+    - [Zonemaster-GUI](installation/zonemaster-gui.md)
+    - [Docker](installation/docker.md)
+- [Upgrade]()
+    - [Backend](upgrade/backend.md)
+        - [v1.0.3](upgrade/backend/upgrade_zonemaster_backend_ver_1.0.3.md)
+        - [v1.1.0](upgrade/backend/upgrade_zonemaster_backend_ver_1.1.0.md)
+        - [v5.0.0](upgrade/backend/upgrade_zonemaster_backend_ver_5.0.0.md)
+        - [v5.0.2](upgrade/backend/upgrade_zonemaster_backend_ver_5.0.2.md)
+        - [v8.0.0](upgrade/backend/upgrade_zonemaster_backend_ver_8.0.0.md)
+        - [v9.0.0](upgrade/backend/upgrade_zonemaster_backend_ver_9.0.0.md)
+- [Configuration]()
+    - [Profiles](configuration/profiles.md)
+    - [Backend](configuration/backend.md)
+        - [Environment Variables](configuration/backend-environment-variables.md)
+    - [GUI](configuration/gui.md)
+- [Using](using/README.md)
+    - [CLI](using/cli.md)
+    - [Backend](using/backend/README.md)
+        - [Getting started](using/backend/getting-started.md)
+        - [RPCAPI Reference](using/backend/rpcapi-reference.md)
+        - [Telemetry](using/backend/telemetry.md)
+    - [GUI](using/gui/README.md)
+        - [API](using/gui/api.md)
```

Comment:
I was expecting a page explaining how to use the graphical user interface from the viewpoint of someone who opens https://zonemaster.fr for example. Having a description of another API here is rather odd.